Shared/Services/SwiftfinNotifications.swift

```diff
@@ -81,6 +81,8 @@ extension Notifications.Key {
     static let didFailMigration = NotificationKey("didFailMigration")
 
     static let itemMetadataDidChange = NotificationKey("itemMetadataDidChange")
```

Comment:
You're correct that this notification was mainly used for the user info since that would change after playback. Now these two notifications mean the same thing and do the same thing. I don't think we should have both, but we can change this notification to send the item instead in the future.

Reply:
For ItemViewModel, is this okay to consolidate [String: String] or Item.Id?

```
    Notifications[.itemMetadataDidChange].publisher
        .sink { [weak self] notification in
            if let userInfo = notification.object as? [String: String] {
                if let itemID = userInfo["itemID"], itemID == item.id {
                    Task { [weak self] in
                        await self?.send(.backgroundRefresh)
                    }
                } else if let seriesID = userInfo["seriesID"], seriesID == item.id {
                    Task { [weak self] in
                        await self?.send(.backgroundRefresh)
                    }
                }
            } else if let itemId = notification.object as? String, itemId == self?.item.id {
                Task { [weak self] in
                    await self?.send(.backgroundRefresh)
                }
            }
        }
        .store(in: &cancellables)
 ```

Reply:
I made the switch. It all seems to work both for userData updates and with metadata updates. The only interaction that's kind of off is if you edit an episode, going back to the season it's incorrect at the EpisodePicker level:

https://github.com/user-attachments/assets/de0c7f51-eff6-4577-8557-15ed07745209